Formula/aws-amplify.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class AwsAmplify < Formula
+  desc "Build full-stack web and mobile apps in hours. Easy to start, easy to scale"
+  homepage "https://aws.amazon.com/amplify"
+  url "https://registry.npmjs.org/@aws-amplify/cli-internal/-/cli-internal-12.0.0.tgz"
+  sha256 "3bad1aad93f0e226cdeed8d742d912a0d149abc0618a4258acb8549205092310"
+  license "Apache-2.0"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Replace universal binaries with native slices.
+    deuniversalize_machos
+  end
+
+  test do
+    amplify_version = shell_output("#{bin}/amplify --version")
+    assert_match version.to_s, amplify_version
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
OK